### PR TITLE
Add Process and Operations meeting

### PR DESCRIPTION
**Pull request by choldgraf**
Every couple of weeks @damianavila and I have been meeting to discuss team processes, systems for coordination and planning, etc. We've been doing this as part of our one-on-one meetings, but now that we have defined "content meetings" series, we felt that it would be better to formalize this type of meeting and make it more accessible to others on the team.

So this PR adds a new type of content meeting called a **Process and Operations** meeting. This PR adds the meeting link and brief overview of the meeting purpose.

### CODE REVIEW

**Comment by jmunroe** [comment]
This is great. I am happy to see the visibility into a "Process and Operations" meeting (even just for information)

**Review by damianavila — APPROVED** [review]

File: meetings/list.md [hunk 1]
```diff
@@ -37,9 +37,23 @@ Covers our high-level organizational mission, values, and principles, our high-l
...
 
+### Process and operations
+
+Covers our operational practices and systems for coordination and planning.
+We discuss ways that we can improve organization-wide practices to become more efficient and effective at doing our work.
+
+It is led by the **`Project Manager`**.
+
+- **Date**: [every other Tuesday](https://calendar.google.com/event?action=TEMPLATE&tmeid=NzRsbm1vc3Y4ZmlmZ2xucDE3cWI0YmxlMmNfMjAyMjEwMjVUMTMwMDAwWiBjXzRoampvdW9qZDhwc3FsOWkxYThuZDF1ZmY0QGc&tmsrc=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com)
+- [ ]
```
Comment (damianavila) [inline meetings/list.md:51]: Why do we have these empty places? I feel you maybe forgot to remove these ones?

File: meetings/list.md [hunk 2]
```diff
@@ -37,9 +37,23 @@ Covers our high-level organizational mission, values, and principles, our high-l
...
+We discuss ways that we can improve organization-wide practices to become more efficient and effective at doing our work.
+
+It is led by the **`Project Manager`**.
+
+- **Date**: [every other Tuesday](https://calendar.google.com/event?action=TEMPLATE&tmeid=NzRsbm1vc3Y4ZmlmZ2xucDE3cWI0YmxlMmNfMjAyMjEwMjVUMTMwMDAwWiBjXzRoampvdW9qZDhwc3FsOWkxYThuZDF1ZmY0QGc&tmsrc=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com)
+- [ ] 
+- [ ] 
+- [ ] Add to team calendar
+- [ ] Document here
+- [ ]
```
Comment (choldgraf) [inline meetings/list.md:55]:
```suggestion
```

**Comment by choldgraf** [comment]
No idea what that was about haha, thanks @damianavila for catching!

**Pull request merged**